app/sidekiq/decision_review/nod_send_email_job.rb

```diff
@@ -15,9 +15,9 @@ class NodSendEmailJob
       user_uuid: nil
     }.freeze
 
-    def perform(email_address, template_id, customisation, line_num)
+    def perform(email_address, template_id, personalisation, line_num)
```

Comment:
Kind of a nit pick here, but personalization (with an z) is the typical US english spelling

Reply:
Yes, I'm using the UK spelling for consistency with VANotify's documentation: https://github.com/department-of-veterans-affairs/vets-api/blob/b1adacea4036c931b490fbf3f82a40959e00464e/modules/va_notify/README.md?plain=1#L42